packages/svelte/src/lib/__testUtil/mockTolgee.ts

```diff
@@ -52,6 +58,30 @@ export const mockTolgee = () => {
   const getLangMock = jest.fn(() => 'mocked-lang');
   const setLangMock = jest.fn();
 
+  const tolgee = {
+    init: () => tolgee as Tolgee,
+    use: () => tolgee as Tolgee,
+    run: runMock.run,
+    stop: stopMock,
+    get lang() {
+      return getLangMock();
+    },
+    set lang(lang: string) {
+      setLangMock(lang);
+    },
+    instant: instantMock,
+    translate: translateMock,
+    onTranslationChange: onTranslationChangeMock,
+    onLangChange: onLangChangeMock,
+    initialLoading: true,
+  } as Partial<Tolgee>;
+
+  const tolgeeClass = jest.fn().mockImplementation(() => tolgee);
+  // @ts-ignore
+  tolgeeClass.init = () => tolgee;
```

Comment:
`(tolgeeClass as any).init` whould be more inormative, what hack are you actually hacking.